app/ui/src/app/integration/import-export/import/integration-import.component.ts

```diff
@@ -91,9 +112,10 @@ export class IntegrationImportComponent implements OnInit, OnDestroy {
                                     response: string,
                                     status: number) => {
       if (status === 200) {
-        this.review = true;
+        this.showButtons = true;
+        this.showReviewStep = !this.checkIfMultiple();
         this.integrationImports$ = JSON.parse(response);
```

Comment:
Should this end with `$`?  It's not an observable is it?  Or is it?  I'm confused :-)

Reply:
This should probably be `this.integrations`, and, yes, not have a `$` at the end.